diff --git a/cmake/XalanChecks.cmake b/cmake/XalanChecks.cmake
index 7f54a2c..b5427ed 100644
--- a/cmake/XalanChecks.cmake
+++ b/cmake/XalanChecks.cmake
@@ -63,7 +63,7 @@ int main()
 XALAN_LINK)
 
 if(NOT XALAN_LINK)
-  message(FATAL_ERROR "Xalan-C++ library link test failed")
+   message(WARNING "Xalan-C++ library link test failed")
 endif(NOT XALAN_LINK)
 
 set(CMAKE_REQUIRED_LIBRARIES ${CMAKE_REQUIRED_LIBRARIES_SAVE})
diff --git a/cmake/XercesChecks.cmake b/cmake/XercesChecks.cmake
index c0559eb..7238dfe 100644
--- a/cmake/XercesChecks.cmake
+++ b/cmake/XercesChecks.cmake
@@ -58,7 +58,7 @@ int main() {
 XERCES_LINK)
 
 if(NOT XERCES_LINK)
-  message(FATAL_ERROR "Xerces-C++ library link test failed")
+   message(WARNING "Xerces-C++ library link test failed")
 endif(NOT XERCES_LINK)
 
 set(CMAKE_REQUIRED_LIBRARIES ${CMAKE_REQUIRED_LIBRARIES_SAVE})
